src/Symfony/Contracts/HttpClient/Exception/HttpExceptionInterface.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Symfony\Contracts\HttpClient\Exception;
+
+use Symfony\Contracts\HttpClient\ResponseInterface;
+
+interface HttpExceptionInterface
```

Comment:
What do you think of `HttpResponseExceptionInterface`? The current name doesn't imply it has anything to do with a response, while the getter implies there's _always_ a response

Reply:
In my opinion the current naming is fine, let's wait for some other comments

Reply:
I'm fine with the name `HttpExceptionInterface` personnaly, but the interface should extend `ExceptionInterface`